sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/api/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.net8.0.cs

```diff
@@ -0,0 +1,257 @@
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents
```

Comment:
Please revert.   You should be generating the API only for your service directory.   No other areas should be included.